tests/Feature/LangTest.php

```diff
@@ -35,12 +41,48 @@ public function testLang(): void
 			$locale = $lang_test->get_locale();
 
 			foreach ($keys as $key) {
-				static::assertArrayHasKey($key, $locale, 'Language ' . $lang_test->code() . ' is incomplete.');
+				try {
+					static::assertArrayHasKey($key, $locale, 'Language ' . $lang_test->code() . ' is incomplete.');
+				} catch (ExpectationFailedException $e) {
+					$msgSection->writeln('<comment>Error:</comment> ' . $e->getMessage());
+					$failed = true;
+				}
 			}
 		}
 
 		static::assertEquals('en', Lang::get_code());
 		static::assertEquals('OK', Lang::get('SUCCESS'));
+		$this->assertFalse($failed);
```

Comment:
Error: Static method 'assertFalse' should not be called dynamically 

```suggestion
		static::assertFalse($failed);
```

Which is also rather obvious compared with the two lines above.